README.md

```diff
@@ -5,10 +5,10 @@
 
 Behold: an experimental barcode format for air-gapped data transfer.
 
-It can sustain speeds of 943+ kilobits/s (~118 KB/s) using just a computer monitor and a smartphone camera!
+It can sustain speeds of 850 kilobits/s (~106 KB/s) using just a computer monitor and a smartphone camera!
```

Comment:
Removing 8-color means we also slow down on our benchmark here. Alas.

But this should be more representative than quoting an 8-color number most people probably couldn't get working -- so I prefer it.